src/scalars/mod.rs

```diff
@@ -0,0 +1,20 @@
+use async_graphql_value::ConstValue;
+
+pub use crate::scalars::email::Email;
+
+mod email;
+
+pub const CUSTOM_SCALARS: &[&str] = &["Email"];
+
+#[derive(schemars::JsonSchema)]
+/// A wrapper to store all custom scalar types
+pub struct Scalars {
+    pub email: Email,
+}
```

Comment:
If we are using `Scalars` primarily for packaging all the scalars into one type, then we should perhaps use an enum.

Reply:
**PLEASE DON'T RESOLVE COMMENTS THAT HAVE NOT BE FIXED**